README.md

```diff
@@ -1,5 +1,14 @@
 # lava-hacks
 a series of useful cli utilities for LAVA server interactions
 
-stream-lava-log:
- usage: python stream-lava-log.py --username <lava username> --token <lava token> --server <http://lava-server/RPC2> --job <lava job id>
\ No newline at end of file
+cat $HOME/.lavarc
+[linaro]
+server: http://lava-server/RPC2
+username: <user>
+token: <auth-token>
+
+ usage: ./stream-lava-log.py --username <lava username> --token <lava token> --server <http://lava-server/RPC2> --job <lava job id>
+ usage: ./stream-lava-log.py --section linaro --job <lava job id>
+
+ # Override config settings on the command line
+ usage: ./stream-lava-log.py --section linaro --token <lava token> --job <lava job id>
```

Comment:
Have a look at the 'Rich Diff' of this change, you'll see it's poorly formatted. Looks like some markdown might be appropriate here:

https://help.github.com/articles/github-flavored-markdown/#fenced-code-blocks